### ISSUE TITLE
Master Wind Waker DLC Not Being Detected?

### ISSUE THREAD

**Issue by Sparky7198** [nedron92/HWL-SaveEditor#5]
(Don't think of this as a priority since i can use the NTR Plugin to get unlimited items in the adventure mode map) I have the master wind waker DLC installed but the save editor dosen't seem to detect it?

**Comment by nedron92** [comment]
Ok? I had no problems with that after all tests I did. 
Could you provide your savefile here, so I can check manually against my one(s). In some circumstances it could be a little bit broken.

**Comment by Sparky7198** [comment]
Sure. il even send a picture of the editor only detecting the other 3 dlc's 

[Save.zip](https://github.com/nedron92/HWL-SaveEditor/files/1759193/Save.zip)
 
https://cdn.discordapp.com/attachments/387788694674997248/417704983170121728/unknown.png

**Comment by Sturtle10** [comment]
yeah, I have the same problem.... Any update on whether its been repaired? if not i'll just have to work with it, which is fine btw.
